### PR TITLE
[MIG] sale_cutoff_time_delivery: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
 stock-logistics-workflow
+partner_tz
```
Comment (sebalix) [inline oca_dependencies.txt]: To remove

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+odoo14-addon-partner-tz @ git+https://github.com/trobz/partner-contact@14.0-mig-partner_tz#subdirectory=setup/partner_tz
```
Comment (sebalix) [inline test-requirements.txt]: Has been merged, we can remove it now
- https://github.com/OCA/partner-contact/pull/1043

**Comment by mmequignon** [comment]
@sebalix @TDu @jgrandguillaume @simahawk @jbaudoux 
As discussed this is one of the modules that may be merged for v14.0.
Here's the thing :
We have 3 modules (`sale_cutoff_time_delivery`, `sale_partner_time_window`, `sale_warehouse_calendar`) that modifies the expected / scheduled dates, and are incompatible with each other because we have to somehow manage the order in which each one overrides `sale.order.line_expected_date` and `sale.order.line._prepare_procurement_values`.
The idea we discussed about is to merge those 3 modules in a single one which should deal with those delivery dates.

File: test-requirements.txt (hunk 2 above)
Comment (dzungtran89) [reply test-requirements.txt]: Hi @sebalix , sorry for late reply, I pushed the update

**Comment by dzungtran89** [comment]
we're checking why the travis failed

**Comment by mmequignon** [comment]
superseded by https://github.com/OCA/sale-workflow/pull/1736

**Pull request closed without merging**